_source/_posts/2023-03-03-step-up-auth.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog_post
+title: "Step-up Authentication in Modern Applications"
+author: indranil-jha
+by: advocate
+communities: [security]
+description: "Understand and apply the principles of step-up authentication"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/step-up-auth/stepup.png
+type: awareness
+---
+
+ Step-up authentication in an application is a pattern of allowing access to non-critical resources using basic level of authentication, and requiring additional authentications for critical resources.
+
+In this article, we will explore the whys and hows of step-up authentication in modern applications that include Single Page Applications (SPA) and native mobile applications.
+
+Federation protocols such as **OpenID Connect (OIDC)** are getting increasingly popular for modern applications to enhance security by trusting an appropriate identity provider. Here we will focus primarily on the **OIDC** standard and how it can be leveraged to provide the step up functions in an idiomatic way.
```

Comment:
```suggestion
Federation protocols such as **OpenID Connect (OIDC)** are getting increasingly popular for modern applications to enhance security by trusting an appropriate identity provider. Here we will focus primarily on the **OIDC** standard and how it can be leveraged to provide the step-up functions in an idiomatic way.
```